AEMCS compilation error in cloud manager

## Required Information

* [ ] AEM Version, including Service Packs, Cumulative Fix Packs, etc: _______

AEM as cloud service, release: 2024.11.18598.20241113T125352Z



* [ ] ACS AEM Commons Version: ___acs-aem-commons-all-6.6.4-cloud.zip
* __
* [ ] Reproducible on Latest? yes

### Expected Behavior
When we run pipeline, we see this error:

[2024/11/25 15:51:44.512] [ERROR] com.adobe.granite.fact.command.CompileJavaCommand - Errors:

apps/acs__002d__commons/components/utilities/genericlist/body__002e__jsp.java 7 Only a type can be imported. com.adobe.acs.commons.mcp.form.impl.GeneratedDialogWrapper resolves to a package

We do install the acs common for cloud service as described in https://adobe-consulting-services.github.io/acs-aem-commons/pages/maven.html

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,9 +10,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 
 ## Unreleased ([details][unreleased changes details])
 
+### Fixed
+
+- #3483 - Fixed issue with genericlist/body.jsp importing a class from an impl package.
+
 ## 6.9.6 - 2024-11-20
 
 ### Fixed
+
 - #3473 - Fix Broken Styling when the notification is active
 - #3474 - Fixed reintroduction of dependency to outdated Apache Commons Collections 3
 
@@ -31,11 +36,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 ## 6.9.0 - 2024-10-29
 
 ### Fixed
+
 - #3459 - Top level properties in parameterized include are now respected.
 - #3460 - Fixes issue where double parameters were not working for the parameterized include
 - #3443 - Content Sync: don't drill down into content tree if recursion is off
 
 ### Changed
+
 - #3385 Made nesting parameterized includes inside a multi-field (ignored resource types) possible
 
 ## 6.8.0 - 2024-10-17

--- a/ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/genericlist/body.jsp
+++ b/ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/genericlist/body.jsp
@@ -17,7 +17,7 @@
   limitations under the License.
   #L%
   --%>
-<%@page import="com.adobe.acs.commons.genericlists.GenericList, com.adobe.acs.commons.mcp.form.impl.GeneratedDialogWrapper"%>
+<%@page import="com.adobe.acs.commons.genericlists.GenericList, com.adobe.acs.commons.mcp.form.GeneratedDialogWrapper"%>
 <%@include file="/libs/foundation/global.jsp"%>
 
 <%

